### PR TITLE
Add cached data endpoint documentation 📚

### PR DESCRIPTION
**Pull request by core-hacked**
# Changes:

- Add documentation for the `latest_vehicle_data` endpoint.
    >The endpoint gets cached data, pushed by the vehicle on sleep, wake and around OTAs.

> REMOVED: - Put the `response` of these into a collapsible element, as they take up too much space.

### CODE REVIEW

File: docs/vehicle/state/data.md [hunk 1]
```diff
@@ -6,7 +6,12 @@ A rollup of all the `data_request` endpoints plus vehicle configuration.
 
 _Note:_ all `*_range` values are in miles, irrespective of GUI configuration.
 
+<details>
```
Comment (timdorr) [inline docs/vehicle/state/data.md:9]: Please don't use details sections. This should be visible without clicking.

Comment (core-hacked) [reply docs/vehicle/state/data.md:9]: The responses are both very large and make it rather unpleasant to read through this part of the documentation, especially on mobile. The response is not as crucial as the endpoint and the request type, so it makes sense for it to be hidden. 

I can revert the change, but it doesn't make much sense from a readability standpoint.

Kind regards.

**Comment by core-hacked** [comment]
Fixed Formatting 🛠️

File: docs/vehicle/state/data.md (hunk 1 above)
Comment (timdorr) [reply docs/vehicle/state/data.md]: There are folks, myself included, that want to see the full response body so we know what keys to look for when parsing out the data. It shouldn't be a separate click away, even if it's long.

**Comment by timdorr** [comment]
Thanks!

**Pull request merged**